src/components/ui/sonner.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTheme } from "next-themes";
+import { Toaster as Sonner } from "sonner";
+
+type ToasterProps = React.ComponentProps<typeof Sonner>;
+
+const Toaster = ({ ...props }: ToasterProps) => {
+  const { theme = "system" } = useTheme();
+
+  return (
+    <Sonner
+      theme={theme as ToasterProps["theme"]}
```

Comment:
Add expand prop. Let's have the notifications expanded by default, while the hover state looks nice, I don't think it will work (esp for mobile screens).